lib/pages/AccountCreation.dart

```diff
@@ -0,0 +1,170 @@
+import 'package:flutter/material.dart';
+import 'package:lend_a_hand/services/AccountService.dart';
+import 'package:email_validator/email_validator.dart';
+
+class AccountCreation extends StatefulWidget {
```

Comment:
Class name is fine, file name should be changed to account_creation.dart

- [ ] Change all other dart files to this convention

Reply:
What convention should we follow for services that consist of only logic?

Reply:
All file names, dir names and packages names (basically anything outside of code itself) should be lower case with "_"